### PR TITLE
Update sidebar for using_eva_da_monitor.md

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
## Description

Update the sidebar menu to link to using_eva_da_monitor.md and maybe fix the block quote errors in using_eva_da_monitor.md's web presentation.

## Dependencies

None

## Impact

None

### CODE REVIEW

**Comment by CoryMartin-NOAA** [comment]
whoops @EdwardSafford-NOAA I merged in @kevindougherty-noaa 's PR first and now yours has conflicts, can you resolve them and then I can review/merge? Or I can attempt to resolve the conflicts this afternoon once I get a chance.

**Comment by EdwardSafford-NOAA** [comment]
No problem.  I'll update it later today.

**Comment by EdwardSafford-NOAA** [comment]
@CoryMartin-NOAA All set -- I've merged in the latest gh-pages changes.

File: docs/_sidebar.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
   - [Installing Eva](installing_eva.md)
   - [Using Eva](using_eva.md)
+  - [using Eva With DA Monitor] (using_eva_DA_monitor.md)
```
Comment (CoryMartin-NOAA) [inline docs/_sidebar.md]: Let's call it EMC DA Monitor

Comment (CoryMartin-NOAA) [reply docs/_sidebar.md]: Just for the sidebar, for clarity

Comment (EdwardSafford-NOAA) [reply docs/_sidebar.md]: So "using Eva with EMC DA Monitor" in the sidebar, right?

Comment (CoryMartin-NOAA) [reply docs/_sidebar.md]: Yeah

Comment (EdwardSafford-NOAA) [reply docs/_sidebar.md]: Done.

**Review by CoryMartin-NOAA — APPROVED** [review]
Thanks!

**Pull request merged**